packages/firestore/src/exp/database.ts

```diff
@@ -149,11 +149,11 @@ export function initializeFirestore(
 }
 
 /**
- * Returns the existing instance of Firestore that is associated with the
+ * Returns the existing `Firestore` instance that is associated with the
```

Comment:
```suggestion
 * Returns the existing {@link Firestore} instance that is associated with the
```

Reply:
And other places where we use \`Firestore\`

Reply:
Done.